f8a_worker/workers/license.py

```diff
@@ -54,24 +49,24 @@ def run_scancode(scan_path):
         result_data = {'status': 'unknown',
                        'summary': {},
                        'details': {}}
-        command = [path.join(getenv('SCANCODE_PATH', '/opt/scancode-toolkit/'),
+        command = [path.join(configuration.SCANCODE_PATH,
```

Comment:
`LicenseCheckTask` inherits from `BaseTask`, so we can use `self.configuration`, can't we ?

Reply:
You are right, but `run_scancode` is a static method so it does not have access to `self`.